README.md

```diff
@@ -23,10 +23,11 @@ Note that Google Tests is pulled in automatically when installing this project a
 
 ## Klepsydra dependencies
 
-* kpsr-core (installed with yaml support)
+* kpsr-core (installed with yaml support)[^1]
 * kpsr-robotics
 * [kpsr-build](https://github.com/klepsydra-technologies/kpsr-build) (automatically pulled)
 
+[^1]: DDS is no longer supported. Please, download `kpsr-core` up to [v7.8.0](https://github.com/klepsydra-technologies/kpsr-core/tree/v7.8.0) to test DDS functionality.
```

Comment:
Do you mean YAML?

Reply:
No, but I think the ticket is a bit outdated since DDS is not mentioned anymore by most of the aforementioned repos and I removed mentions to it from this repo today.